pom.xml

```diff
@@ -8,7 +8,7 @@
 
   <groupId>org.csanchez.jenkins.plugins</groupId>
   <artifactId>kubernetes</artifactId>
-  <version>${revision}${changelist}</version>
+  <version>${revision}${changelist}-test</version>
```

Comment:
You should not need to do this even temporarily. See [JEP-305](https://jenkins.io/jep/305).